app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -121,32 +140,46 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_voting_page);
         overridePendingTransition(R.anim.slide_in_right, R.anim.slide_out_left);
 
+        NetworkStateReceiverListener networkStateReceiverListener =
```

Comment:
Maybe it’s better to extract the code handling the connectivity in a separated method with a meaningful name

Reply:
In fact, it is used in several classes. Is it possible to wrap it directly in an another utility class, to avoid code duplication?